src/styles/global.scss

```diff
@@ -14,6 +14,86 @@
   }
 }
 
+html.is-extension {
+  width: $extension-width;
+  height: $extension-height;
+
+  // fixes issue with extension not rendering properly
+  ion-tabs,
+  .tabs-inner,
+  ion-router-outlet,
+  .ion-page {
+    contain: layout style !important;
+  }
+
+  body {
+    position: unset;
+  }
+}
+
+// When scanner is active we need to hide everything else
+// because the scanner will show-up behind the webview
+body.scanner-active {
+  background-color: transparent;
+
+  --background: transparent;
+  --ion-background-color: transparent;
+  --screen-bg-color: transparent;
+
+  .app-wrapper {
+    --screen-bg-color: transparent;
+
+    background: transparent;
+  }
+
+  .ionic-wrapper {
+    background-color: transparent;
+  }
+
+  #app {
+    background: transparent;
+    position: absolute;
+    z-index: 2;
+    width: 100%;
+  }
+
+  ion-content,
+  .ion-padding {
+    --background: transparent;
+
+    background: transparent;
+  }
+}
+
+.account-details .ion-padding,
+.account-details-names .ion-padding {
+  background-color: $color-bg-4;
+}
+
+ion-content {
+  /** hide scrollbar **/
+  --offset-bottom: auto !important;
+  --overflow: hidden;
+
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  /** remove left-right padding **/
+  &::part(scroll) {
+    --padding-end: 0;
+    --padding-start: 0;
+    --padding-top: 0;
+  }
+}
+
+/** temporary solution for adding background-color to pages **/
+.ion-padding {
```

Comment:
Why the name is `padding` if the styles are related to background? It makes no sense.